src/state/middleware/scigateway.middleware.tsx

```diff
@@ -103,6 +103,17 @@ export const listenToPlugins = (
                 scigateway: { startUrl: pluginMessage.detail.payload.link },
               })
             );
+          } else if (
```

Comment:
Will be refactored if we decide "homepageUrl" will behave differently to "startUrl". Besides, this code only ensures that the URL is navigated to when the app is first launched and doesn't handle any future redirects.